### DISCUSSION TITLE
Reading Material

### DISCUSSION THREAD

**Discussion by 1cg** [bigskysoftware/missing#6] [Ideas]
Please add reading material to this discussion

**Comment by fractaledmind** [comment]
Demo document with all HTML5 elements:
* https://github.com/dbox/html5-kitchen-sink

Under-engineered HTML components:
* https://adrianroselli.com/2017/05/under-engineered-custom-radio-buttons-and-checkboxen.html
* https://adrianroselli.com/2019/03/under-engineered-toggles.html
* https://adrianroselli.com/2019/08/under-engineered-toggles-too.html
* https://adrianroselli.com/2019/09/under-engineered-text-boxen.html
* https://adrianroselli.com/2020/11/under-engineered-responsive-tables.html
* https://adrianroselli.com/2021/03/under-engineered-select-menus.html
* https://adrianroselli.com/2021/12/under-engineered-dependency-questions.html 
* https://www.scottohara.me/blog/2019/07/10/the-output-element.html

Tailwind Plugins for default styling:
* https://play.tailwindcss.com/uj1vGACRJA?layout=preview
* https://tailwindcss-forms.vercel.app/kitchen-sink.html

Example of custom elements with only CSS:
* https://almonk.github.io/pylon/

`:where` selector
* https://developer.mozilla.org/en-US/docs/Web/CSS/:where
* https://caniuse.com/mdn-css_selectors_where
